src/plugins/vis_type_vislib/public/vislib/percentage_mode_transform.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+// @ts-ignore
+import numeral from '@elastic/numeral';
+
+const numeralInst = numeral();
+
+export function getValueForPercentageMode(value: string | number, percentageFormatPattern: string) {
```

Comment:
please explain. Can we use `FieldsFormats` instead?

Reply:
Yes, I checked it and we can use FieldsFormats. Now I use your proposal. Thank you!